Makefile

```diff
@@ -25,12 +25,22 @@ ENVIRONMENT_FILE := pdk_regression.yml
 # Path to regression
 KLAYOUT_TESTS := klayout/drc/testing/
 
+
 include third_party/make-env/conda.mk
 
 # Lint python code
-lint: | $(CONDA_ENV_PYTHON)
+lint_python: | $(CONDA_ENV_PYTHON)
 	@$(IN_CONDA_ENV) flake8 .
 
+# Lint ruby code
+lint_ruby:| $(CONDA_ENV_PYTHON)
```

Comment:
should we add `ruby` as a dependencies of the conda env?

Reply:
Same as below